components/xsd-fu/python/ome/modeltools/property.py

```diff
@@ -345,6 +351,50 @@ def _get_unitsType(self):
         _get_unitsType,
         doc="""The property's units type.""")
 
+    def _get_enumProperties(self):
+        if self.isEnumeration:
+            if self.enumProps is None:
+                self.enumProps = dict()
+
+                try:
+                    root = self.model.schemas['ome']
+                    enum = root.find("{http://www.w3.org/2001/XMLSchema}simpleType[@name='%s']" % self.langType)
```

Comment:
Yes, this is quite annoying.  I was about to make a card for this.  A wider question would be whether we should or could drop Python 1.6 support for the bioformats build.  We could switch the jobs to centos 7 nodes or use Python 2.7 on CentOS 6.